src/Model/Resolver/Aggregations.php

```diff
@@ -13,11 +13,35 @@
 namespace ScandiPWA\CatalogGraphQl\Model\Resolver;
 
 use Magento\CatalogGraphQl\Model\Resolver\Aggregations as AggregationsBase;
+use Magento\CatalogGraphQl\Model\Resolver\Layer\DataProvider\Filters;
+use Magento\CatalogGraphQl\DataProvider\Product\LayeredNavigation\LayerBuilder;
 use Magento\Framework\GraphQl\Config\Element\Field;
 use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Framework\Exception\LocalizedException;
 use Magento\Store\Api\Data\StoreInterface;
+use Magento\Catalog\Model\ResourceModel\Eav\Attribute;
 
 class Aggregations extends AggregationsBase {
+
+    /**
+     * @var Attribute
+     */
+    private Attribute $attribute;
```

Comment:
Please don't use property types, since php 7.3 doesn't support it and it makes deploy to fail on some projects that haven't updated their php.